### PR TITLE
Add almosthere

### PR DESCRIPTION
**Pull request by horta**

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,79 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml]: Please do not delete the `example` recipe.

Comment (horta) [reply recipes/example/meta.yaml]: Ops, sorry!

**Comment by horta** [comment]
I have added the option to skip macos build. Would it be possible to merge it once the circleci and appveyor pass?

**Comment by horta** [comment]
All checks pass \o/

**Comment by mariusvniekerk** [comment]
ping @conda-forge/staged-recipes

**Pull request merged**